integration/src/main/java/scripts/testscripts/GetProviderPassport.java

```diff
@@ -0,0 +1,43 @@
+package scripts.testscripts;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import bio.terra.externalcreds.api.OidcApi;
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import com.google.api.client.http.HttpStatusCodes;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import scripts.utils.ClientTestUtils;
+
+@Slf4j
+public class GetProviderPassport extends TestScript {
+  private String provider;
+  private OidcApi oidcApi;
+
+  @Override
+  public void setup(List<TestUserSpecification> testUsers) throws Exception {
+    // TODO: do we want to loop through this list in case we want multiple test users in the future?
```

Comment:
I was wondering about this too. @dvoet do you have any preference on whether we use more than one test user to authenticate for these tests? I am currently only adding the one actual Sam account to our generated data but I could add in more if we wanted.